components/com_content/tmpl/form/edit.php

```diff
@@ -48,106 +48,97 @@
-                <?php echo $this->form->renderField('access'); ?>
-                <?php if (is_null($this->item->id)) : ?>
+                <?php if ($params->get('show_urls_images_frontend')) : ?>
+                    <?php echo HTMLHelper::_('uitab.addTab', $this->tab_name, 'images', Text::_('COM_CONTENT_IMAGES_AND_URLS')); ?>
+                    <?php echo $this->form->renderField('image_intro', 'images'); ?>
+                    <?php echo $this->form->renderField('image_intro_alt', 'images'); ?>
+                    <?php echo $this->form->renderField('image_intro_alt_empty', 'images'); ?>
+                    <?php echo $this->form->renderField('image_intro_caption', 'images'); ?>
+                    <?php echo $this->form->renderField('float_intro', 'images'); ?>
+                    <?php echo $this->form->renderField('image_fulltext', 'images'); ?>
+                    <?php echo $this->form->renderField('image_fulltext_alt', 'images'); ?>
+                    <?php echo $this->form->renderField('image_fulltext_alt_empty', 'images'); ?>
+                    <?php echo $this->form->renderField('image_fulltext_caption', 'images'); ?>
+                    <?php echo $this->form->renderField('float_fulltext', 'images'); ?>
+                    <?php echo $this->form->renderField('urla', 'urls'); ?>
+                    <?php echo $this->form->renderField('urlatext', 'urls'); ?>
+                    <div class="control-group">
+                        <div class="controls">
+                            <?php echo $this->form->getInput('targeta', 'urls'); ?>
+                        </div>
+                    </div>
+                    <?php echo $this->form->renderField('urlb', 'urls'); ?>
+                    <?php echo $this->form->renderField('urlbtext', 'urls'); ?>
                     <div class="control-group">
-                        <div class="control-label">
+                        <div class="controls">
+                            <?php echo $this->form->getInput('targetb', 'urls'); ?>
                         </div>
+                    </div>
+                    <?php echo $this->form->renderField('urlc', 'urls'); ?>
+                    <?php echo $this->form->renderField('urlctext', 'urls'); ?>
+                    <div class="control-group">
                         <div class="controls">
-                            <?php echo Text::_('COM_CONTENT_ORDERING'); ?>
+                            <?php echo $this->form->getInput('targetc', 'urls'); ?>
                         </div>
                     </div>
+                    <?php echo HTMLHelper::_('uitab.endTab'); ?>
                 <?php endif; ?>
-            <?php echo HTMLHelper::_('uitab.endTab'); ?>
 
-            <?php if (Multilanguage::isEnabled()) : ?>
-                <?php echo HTMLHelper::_('uitab.addTab', $this->tab_name, 'language', Text::_('JFIELD_LANGUAGE_LABEL')); ?>
+				<?php echo LayoutHelper::render('joomla.edit.params', $this); ?>
```

Comment:
```suggestion
                <?php echo LayoutHelper::render('joomla.edit.params', $this); ?>
```
Fix PHPCS (spaces and not tabs for indentation)